src/CashuWallet.ts

```diff
@@ -299,8 +314,13 @@ class CashuWallet {
 			keysetId?: string;
 			offline?: boolean;
 			includeFees?: boolean;
+			includeDleq?: boolean;
 		}
 	): Promise<SendResponse> {
+		if (options?.includeDleq ?? false) {
+			// only pick the ones with a DLEQ proof
+			proofs = proofs.filter((p: Proof) => p.dleq != undefined);
```

Comment:
If `includeDleq` controls wether or not only proofs with a dleq are considered for sending I would prefer a name like `requireDleq`. What do you think?